.travis.yml

```diff
@@ -96,24 +97,21 @@ _nodejs_deploy_template: &_nodejs_deploy_template
       cargo: true
       directories:
           - node_modules
-  before_script:
-      - yarn install --ignore-scripts
   script:
-      - cd nodejs_container
-      - node publish.js
+      - ./scripts/build_nodejs_container.sh
   # deploy the node-pre-gyp binary to github releases so it's there when the npm package is installed
   deploy:
       provider: releases
       api_key:
-          secure: BIgLicKCn7lwSuAtaZeq514Tqu7YVMAHi8N/R7MfZc1NbzFs40q5MyNMf1krVoXwJF3BoQ8Y5pb2iesqsrZLCv9lWcrOeOjsTywabZnTan0DcDZs5TimyNknCNSkF4OoXjIWefkEdtB8PzkC5zNL/P2nIMyiuHWHuQYiZGhclwO6x0LAj3aYLrdkVeR0129Rdh05Np5edMY4DHfq9tLmQ/IfyE7U97ttAwXHIKnZnQ+SRxgYxSGaIDSHLA3Ah4HHKl7KUgh6wDo19zoT5jdisI/bDO5YCgUcRAphYKrTebA4GiBIUWikg7eCUQmLj98DJZFgEonmKrR+MIYXJ4u47lvrAcC0T/1+Alo6oAHA9j23Yf737LpWnkLcxxtpgp+ykbGjG42kW2Okoj5yyqPO6wGxr808yaRORQDfp/7n4cxr+pSQIM4eIaUySakNtpYDwWABhYje4MkQI6s1iLRbCkK85DiUA3Ptmvpwbyhz7pbXadNQ1chydHoGvD1qGK1BCfRjTmfUlA/UzJeyq9pnMI8RqyVQmjDT8MpFR+QfOKGUDyvOBFGAtvR5TdJT20LEcXDEX7O+F4apZXhmG7dFMpHKIXbtSJyNFYeM+0cJCQUoJHp9Zvk5yJy8cf1VJDY2v75QzaQ/Aodz4M9h7WVnK8tWNqDKETDs7e7KUQTDn7A=
+          secure: SfAyYHw8iC74CY8SrSnEsA0GPLxpCwguYziw19ZlBz6Nq+HxqNWoexw87TdrTIkxUKJpNxoLzqeKmigpAzr8Q4Y8j72tD5JyTSLwTnjChclM1BMRh7NR3dL8gTrBRuRZ4ALJr4EP1sxXI16W3lEz7hrS4tyws2e0W3xb80Os7p9fNQHpKCAncON622mDSznPbUXbUCNdosqZq+dZua6RZ9xmMaba1La3+vZn+hzvcNvI2mtiK/KOIDbo4uzZEUf7URtKmtkslilIc+3568iuCjPl5zV1s+/VN2/SEuxcxuVtxXUV6ZVguniF0z4L5YOCwnnadUJtlXYxxQznRu/YT7xvlvitSNulDm81jACiwKZ7LbbXssfzIb+gTBXiTdFyUDJUEHL8U992K7oDJN2nM6jPsqnmjuqalFcV/raVmpZA/gA3eCqyCw8is363NqEFpGgczBnL84h81x63RrXsXb6oZq2E5fQkIHP/4srG/s359achguHmaf/g69NiAnhCz9P2xF15zFqRPZnUA7q9ONpSAJrkOWVxBEntfIsLL6P07y3rCdXhs6A6Q5lh5YcWdOH2zEtsVXaI2/AbVgB4g0CWChts6tpqrNa5QzRIF82LLjhQTZOkpBUwMliunzmHMhIz0ojHMjay0JrCuYF2lbUfQ/f3S4Clnz19Z5DPo/A=
```

Comment:
When using travis-ci.org, you have to run
`travis setup releases --org`

Then what I do is copy out the resulting `secure` prop from the file, reset the branch, and then copy the new `secure` prop back in because I don't want to overwrite the rest of the configuration